tests/config/Earthfile

```diff
@@ -0,0 +1,35 @@
+VERSION 0.6
+
+FROM ..+base
+
+test:
+    COPY expected-*.yml .
+    RUN touch config.yml
```

Comment:
I wonder if we should include a

```
RUN earthly --config config.yml config global.disable_analytics true 2>&1 | grep 'failed to read from config.yml: open config.yml: no such file or directory'
```

to ensure earthly doesn't create overridden config files when they don't exist (which makes it more obvious when someone overrides a config file but typos it and points the the wrong location).